crates/protocol/src/block.rs

```diff
@@ -78,7 +78,7 @@ pub struct L2BlockInfo {
     pub seq_num: u64,
 }
 
-#[cfg(any(test, feature = "arbitrary"))]
+#[cfg(feature = "arbitrary")]
```

Comment:
oh yeah, I want to do this for more crates as well because this has been driving me insane lately